src/core/stream_send.c

```diff
@@ -1513,6 +1597,37 @@ QuicStreamOnResetAck(
     }
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicStreamOnResetReliableAck(
+    _In_ QUIC_STREAM* Stream
+    )
+{
+    QuicTraceLogStreamVerbose(
+        ResetReliableAck,
+        Stream,
+        "Reset Reliable ACKed in OnResetReliableAck. Send side. UnAckedOffset=%llu, ReliableOffsetSend=%llu",
+        Stream->UnAckedOffset, Stream->ReliableOffsetSend);
+
+    if (Stream->UnAckedOffset >= Stream->ReliableOffsetSend && !Stream->Flags.LocalCloseAcked) {
```

Comment:
Can we add an assert that the stream hasn't been somehow reset already too? I'm just wondering if this check for `LocalCloseAcked` isn't enough. What if the peer reset their receive before we could get an ACK for reliable reset on send?

Reply:
Resolved from discussions